frontend/tests/integration/components/transportation/tdf/modal-splits-test.js

```diff
@@ -57,6 +57,33 @@ module('Integration | Component | transportation/tdf/modal-splits', function(hoo
       {{/transportation/tdf/modal-splits}}
     `);
 
-    assert.ok(this.element);
+    // if manualModeSplits is true, then modeSplits = modeSplitsFromUser, which are all default 0
+    assert.equal(this.element.querySelector('[data-test-total-am]').textContent, '0', 'total am default');
+    assert.equal(this.element.querySelector('[data-test-total-md]').textContent, '0', 'total md default');
+    assert.equal(this.element.querySelector('[data-test-total-pm]').textContent, '0', 'total pm default');
+    assert.equal(this.element.querySelector('[data-test-total-saturday]').textContent, '0', 'total saturday default');
+
+    // USER FILLS IN MANUAL MODE SPLITS
+    // total for am should be 3
+    await fillIn('[data-test-modal-split-input-am="auto"]', 1);
+    await fillIn('[data-test-modal-split-input-am="taxi"]', 2);
+
+    // total for md should be 7
+    await fillIn('[data-test-modal-split-input-md="bus"]', 3);
+    await fillIn('[data-test-modal-split-input-md="subway"]', 4);
+
+    // total for pm should be 11
+    await fillIn('[data-test-modal-split-input-pm="railroad"]', 5);
+    await fillIn('[data-test-modal-split-input-pm="auto"]', 6);
+
+    // total for sat should be 15
+    await fillIn('[data-test-modal-split-input-saturday="taxi"]', 7);
+    await fillIn('[data-test-modal-split-input-saturday="bus"]', 8);
+
+    // tests for total.am, total.md, total.pm, total.saturday in computed property `total`
+    assert.equal(this.element.querySelector('[data-test-total-am]').textContent, '3', 'total am calculated');
+    assert.equal(this.element.querySelector('[data-test-total-md]').textContent, '7', 'total md calculated');
+    assert.equal(this.element.querySelector('[data-test-total-pm]').textContent, '11', 'total pm calculated');
+    assert.equal(this.element.querySelector('[data-test-total-saturday]').textContent, '15', 'total saturday calculated');
```

Comment:
@trbmcginnis this PR looks super clean to me, not fully knowing yet how to write tests I was able to decipher exactly what you're doing here - thanks to your code organization, small/discrete PR, and excellent code comments.  Thank you!

Reply:
:) :)